src/Widgets/FactWidget/FactWidgetEditingConfig.js

```diff
@@ -8,4 +8,18 @@ Scrivito.provideEditingConfig("FactWidget", {
     key: "Lorem ipsum",
     value: "12",
   },
+  validations: [
+    [
+      "value",
+
+      (value, { widget }) => {
+        if (!widget.get("key") && !value) {
+          return {
+            message: "One, key or value, must be set.",
```

Comment:
```suggestion
            message: "The key or value (or both) must be set."
```

Reply:
Don't forget the trailing `,`:

```suggestion
            message: "The key or value (or both) must be set.",
```